src/components/CharacterForm/CharacterForm.js

```diff
@@ -138,7 +143,9 @@ const CharacterForm = ({ setOpen }) => {
         className={classes.title}
         color="text.disabled"
       >
-        Add New Character
+        {params && params[0] === formModeEnum.edit
+          ? `Edit Character - ${params[1]}`
+          : "Add New Character"}
```

Comment:
You should be defining a variable for this called `isEdit` (or similar) so your form will have the ability to to do all sorts of conditional stuff depending on the mode you are in. You will be able to reuse that variable instead of repeatedly writing this logic